app/views/patients/_call_log.html.erb

```diff
@@ -2,7 +2,7 @@
   <div class="margin-bottom col-sm-12">
     <h2>
       Call Log
-      <small><%= link_to "Record new call <span class='glyphicon glyphicon-earphone' aria-hidden='true'></span><span class='sr-only'>Call</span>".html_safe, "#call-#{@patient.primary_phone_display}", data: {toggle: "modal"} %></small>
+      <span class="pull-right new-call-entry"><small><%= link_to "Record new call".html_safe, "#call-#{@patient.primary_phone_display}", data: {toggle: "modal"} %></small></span>
```

Comment:
nice! I think we can remove the `html_safe` here -- it was there so we could get the glyphicon in, and if the glyphicon's coming out then we can ditch the `html_safe` too.